### PR TITLE
feat: 🎸 header and footer

### PR DESCRIPTION
**Pull request by hieunguyenebox**

### CODE REVIEW

**Comment by hieunguyenebox** [comment]
@WeiLi512  plz review

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
     "serve": "11.3.2",
     "shelljs": "0.8.4",
     "styled-components": "5.1.0",
+    "styled-components-breakpoint": "3.0.0-preview.20",
```
Comment (WeiLi512) [inline package.json:78]: Where did you use this module?

File: src/assets/images/logo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....204c-2.168-7.505-9.095-13-17.295-13-9.934 0-18 8.066-18 18s8.066 18 18 18c8.2 0 15.127-5.495 17.295-13z" transform="translate(-1 224.497207)"/><path d="m403 39.842-36-39.982-.156.14h-6.844v50c3.863 0 7-3.137 7-7v-32.679l36 39.982.337-.303h6.663v-50c-3.863 0-7 3.137-7 7z" transform="translate(-1 224.637328)"/><path d="m90.208 50 24.83-50.908 3.867 7.928-.008.016 20.955 42.964h-7.788l-2.439-5h-29.244l-2.439 5zm12.611-10h24.368l-12.184-24.98z" transform="translate(178.792 225.405)"/><path d="m180 50h50c0-2.76-2.24-5-5-5h-38v-38c0-3.863-3.137-7-7-7z" transform="translate(-1 224.497207)"/><path d="m90.208 50 24.83-50.908 3.867 7.928-.008.016 20.955 42.964h-7.788l-2.439-5h-29.244l-2.439 5zm12.611-10h24.368l-12.184-24.98z" transform="translate(-.208447 225.405)"/><path d="m37.558 25h-.109c6.927 0 12.551-5.624 12.551-12.551v-.007c0-6.867-5.575-12.442-12.442-12.442h-37.558v50h37.449c6.927 0 12.551-5.624 12.551-12.551v-.007c0-6.867-5.575-12.442-12.442-12.442zm5.442 11.526c0-4.706-3.82-8.526-8.526-8.526h-27.474v17h27.526c4.677 0 8.474-3.797 8.474-8.474zm0-22.499c0-4.982-4.045-9.027-9.027-9.027h-26.973v18h27.027c4.952 0 8.973-4.021 8.973-8.973z" transform="translate(0 224.497207)"/></g></svg>
```
Comment (WeiLi512) [inline src/assets/images/logo.svg:1]: Please move all svg icons to 'assets/icons' directory.

File: src/components/uikit/AnimatedLink.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
```
Comment (WeiLi512) [inline src/components/uikit/AnimatedLink.tsx:1]: Just put the UI components in the `components` directory for now.

**Review by WeiLi512 — CHANGES_REQUESTED** [review]
Please make UI the same as the prototype. If then, please request to review.

**Comment by hieunguyenebox** [comment]
@WeiLi512  plz check again

**Review by parrot9design — CHANGES_REQUESTED** [review]
### Header

**Current** | [**Design**](http://balanced-development.s3-website-us-east-1.amazonaws.com/stats/)
----------- | -------------
Screen Shot 2021-06-22 at 8 14 01 PM | Screen Shot 2021-06-22 at 8 14 07 PM

- Give the logo a width of 100px
- Make the navigation items 16px
- Include the page title `Statistics`
- Remove `Overview` from the first section
-  Fix the transparency of the total value locked icon



### Footer

**Current** | [**Design**](http://balanced-development.s3-website-us-east-1.amazonaws.com/stats/)
----------- | -------------
Screen Shot 2021-06-22 at 8 21 49 PM | Screen Shot 2021-06-22 at 8 21 59 PM

- Add the divider line 
- Give the logo a width of 100px
- Make the navigation items 16px

**Comment by parrot9design** [comment]
The logo and navigation sizing looks much better. 👍 

The overview and governance sections still need a few adjustments:

**Current** | [**Design**](http://balanced-development.s3-website-us-east-1.amazonaws.com/stats/)
----------- | -------------
Screen Shot 2021-06-24 at 3 00 37 PM | Screen Shot 2021-06-22 at 8 14 07 PM

- Include the page title `Statistics`
- Remove the `Overview` heading
- Fix the transparency of the total value locked icon
- Remove the bold style from the values (should be a normal weight)



### Footer

**Current** | [**Design**](http://balanced-development.s3-website-us-east-1.amazonaws.com/stats/)
----------- | -------------
Screen Shot 2021-06-24 at 3 00 44 PM | Screen Shot 2021-06-24 at 3 00 51 PM


- Remove the bold style from the values (should be a normal weight)

**Comment by parrot9design** [comment]
To recap:

1. The transparency of the TVL icon needs fixed (I believe it’s isolated to this PR)
2. The overview and governance stats need the bold style removed
3. We need to add `Stats` to the navigation:

Top of page
3F5AB1C8-C3B2-4717-BC87-763CBEC8DEDC

Bottom of page
A394CD3F-2002-48CB-93A1-108E79E55778

@WeiLi512 @hieunguyenebox Let’s make sure this PR makes it into tomorrow’s release.

**Comment by hieunguyenebox** [comment]
@parrot9design all fixed, please check

**Comment by parrot9design** [comment]
Looks good! Just need to make the underline active for the stats link so people know it's the active page.

Example:

Screen Shot 2021-06-27 at 9 01 29 AM
Screen Shot 2021-06-27 at 9 01 26 AM

File: src/sections/PairSection.tsx [hunk 4]
```diff
@@ -61,53 +64,54 @@ export default function PairSection() {
...
-                <TotalIcon />
-                <Text ml={2}>Total</Text>
-              </Flex>
-            </FooterText>
-            <FooterText>-</FooterText>
-            <FooterText>{getFormattedNumber(total.participant, 'number')}</FooterText>
-            <FooterText>{getFormattedNumber(total.tvl, 'currency0')}</FooterText>
-            <FooterText>{getFormattedNumber(total.volume, 'currency0')}</FooterText>
-            <FooterText>{getFormattedNumber(total.fees, 'currency0')}</FooterText>
+      <Box overflow="scroll">
```
Comment (WeiLi512) [inline src/sections/PairSection.tsx:67]: Instead of wrapping with `Box`, please customize `List` component.

File: src/sections/TokenSection.tsx [hunk 5]
```diff
@@ -55,51 +60,52 @@ export default function TokenSection() {
       <Typography variant="h2" mb={5}>
         Token
       </Typography>
+      <Box sx={{ overflow: 'scroll' }}>
```
Comment (WeiLi512) [inline src/sections/TokenSection.tsx:63]: Instead of wrapping with `Box`, please customize `List` component.

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

**Pull request merged**